Fixed none-working code samples.

Hupka opened the pull request.

Hupka committed:
Fixed none-working code samples.

FabianLars commented on docs/guides/plugin.md:
````diff
@@ -51,15 +51,19 @@ Using the `tauri::plugin::Builder` you can define plugins similar to how you def
 
 ```rust
 use tauri::plugin::{Builder, TauriPlugin};
````
Can you add the `Runtime` import too?

Hupka replied:
On it!

Hupka committed:
Update plugin.md

Added correct imports, fixed a naming issue. Both snippets were tested compiling.

FabianLars commented:
Last thing, can you revert to 2-spaces indentation? (i don't like it either, but it needs to be consistent)

Hupka committed:
Update plugin.md
